### ISSUE TITLE
print statement as invalid syntax

### ISSUE THREAD

**Issue by TadLeonard** [timothycrosley/deprecated.frosted#1]
The `print` statement is flagged as a syntax error. Is `frosted` only intended for py3k? Do I need to do something with `~/.frosted.cfg`?

Three files. The first uses the Python2 `print` statement. The second uses the Python3 `print()` function (though it's really just using the statement + object-in-parentheses in my case). The third is a syntax error for any version of Python.

``` sh
$ tail print*
==> print2.py <==
#!/usr/bin/env python

print "Hello, Frosted"

==> print3.py <==
#!/usr/bin/env python

print("Hello, Frosted")

==> printbroken.py <==
#!/usr/bin/env python

this is not valid Python for any version of the language
```

They work as expected.

``` sh
$ ./print2.py 
Hello, Frosted
$ ./print3.py 
Hello, Frosted
$ ./printbroken.py 
  File "./printbroken.py", line 3
    this is not valid Python for any version of the language
                           ^
SyntaxError: invalid syntax
```

`frosted` chokes on the first, passes the second, and correctly flags a syntax error for the third file.

``` sh
$ frosted print2.py 
print2.py:3:21: invalid syntax
print "Hello, Frosted"
                     ^
$ frosted print3.py 
$ frosted printbroken.py 
printbroken.py:3:23: invalid syntax
this is not valid Python for any version of the language
                       ^
```

`pyflakes` behaves like I'd expect for each file.

``` sh
$ pyflakes print2.py 
$ pyflakes print3.py 
$ pyflakes printbroken.py 
printbroken.py:3: invalid syntax
this is not valid Python for any version of the language
                        ^
```

Hopefully this is enough information.

``` sh
$ python --version
Python 2.7.1
$ frosted --version
frosted 1.0.0
$ pyflakes --version
0.7.3
```

**Comment by timothycrosley** [comment]
Hi @TadLeonard  thanks for reporting! Frosted is intended for Python2.6+ and I will make a new release soon to address this issue.

**Comment by timothycrosley** [comment]
Fixed in hot-fix release 1.0.1, thanks again for reporting!

**Comment by TadLeonard** [comment]
Looks like it was an easy fix! I guess those `__future__` monkey patches are kinda disruptive for a project like this. Thanks for the quick response.

``` sh
$ frosted print2.py 
$ frosted --version
frosted 1.0.1
```
